astropy/io/fits/diff.py

```diff
@@ -29,7 +29,7 @@
 from .card import Card, BLANK_CARD
 from .header import Header
 # HDUList is used in one of the doctests
-from .hdu.hdulist import fitsopen, HDUList  # pylint: disable=W0611
```

Comment:
On the second thought I can't read, and only see the big red lines for unused imports. Removing ``HDUList`` made the doctests fail. 
@embray - Should I open another PR to add it back, or you prefer cherry-picking https://github.com/bsipocz/astropy/commit/7035a013ed2ed9734d9432f3fc6763366a51e0d9 ? I'm sorry for not spotting it locally.